gunicorn/glogging.py

```diff
@@ -318,18 +318,18 @@ def atoms(self, resp, req, environ, request_time):
         if hasattr(req_headers, "items"):
             req_headers = req_headers.items()
 
-        atoms.update(dict([("{%s}i" % k.lower(), v) for k, v in req_headers]))
+        atoms.update({"{%s}i" % k.lower(): v for k, v in req_headers})
```

Comment:
We still need to support Python 2.6, I'd add consider-using-dict-comprehension to .pylintrc with a TODO comment to remind us that we can use dict comprehensions once we drop Python 2.6 support.

Reply:
sounds good